src/languageHandler.tsx

```diff
@@ -88,8 +88,25 @@ export function _td(s: string): string { // eslint-disable-line @typescript-esli
  * */
 const translateWithFallback = (text: string, options?: object): { translated?: string, isFallback?: boolean } => {
     const translated = counterpart.translate(text, { ...options, fallbackLocale: counterpart.getLocale() });
-    if (!translated || /^missing translation:/.test(translated)) {
+    if (!translated || translated.startsWith("missing translation:")) {
         const fallbackTranslated = counterpart.translate(text, { ...options, locale: FALLBACK_LOCALE });
+        if (!fallbackTranslated || fallbackTranslated.startsWith("missing translation:")
+            && process.env.NODE_ENV !== "development") {
```

Comment:
```suggestion
        if (!fallbackTranslated ||
            (fallbackTranslated.startsWith("missing translation:") && process.env.NODE_ENV !== "development")
        ) {
```

To save someone needing to check up the operator precedence